### ISSUE TITLE
UICanvas issue

### ISSUE THREAD

**Issue by Tryibion** [FlaxEngine/FlaxEngine#1108]
**Issue description:**
When I have UI Canvas in world screen space render mode. I get this error sometimes in the prefab editor and in the world when the prefab is spawned. It causes the UI not work until the editor is restarted.

Prefab setup:
![image](https://github.com/FlaxEngine/FlaxEngine/assets/71274967/bbee2795-f662-43e5-a36a-112c431607ef)

```
[ 00:17:53.268 ]: [Warning] Exception has been thrown during UICanvas::PostDeserialize. Invalid ManagedHandle
Stack strace:
   at FlaxEngine.Interop.ManagedHandle.ManagedHandlePool.GetObject(IntPtr handle) in Z:\Flax\Source\FlaxEngine\Source\Engine\Engine\NativeInterop.Managed.cs:line 502
   at FlaxEngine.Interop.ManagedHandle.get_Target() in Z:\Flax\Source\FlaxEngine\Source\Engine\Engine\NativeInterop.Managed.cs:line 349
   at FlaxEngine.Interop.NativeInterop.GetObjectType(ManagedHandle handle) in Z:\Flax\Source\FlaxEngine\Source\Engine\Engine\NativeInterop.Unmanaged.cs:line 574
   at FlaxEngine.Object.<Internal_Create1>g____PInvoke|28_0(IntPtr type)
   at FlaxEngine.Object.Internal_Create1(Type type) in Z:\Flax\Source\FlaxEngine\Microsoft.Interop.LibraryImportGenerator\Microsoft.Interop.LibraryImportGenerator\LibraryImports.g.cs:line 53648
   at FlaxEngine.Object.New[T]() in Z:\Flax\Source\FlaxEngine\Source\Engine\Scripting\Object.cs:line 109
   at FlaxEngine.UICanvas.Setup() in Z:\Flax\Source\FlaxEngine\Source\Engine\UI\UICanvas.cs:line 529
   at FlaxEngine.UICanvas.PostDeserialize() in Z:\Flax\Source\FlaxEngine\Source\Engine\UI\UICanvas.cs:line 752
   at FlaxEngine.Interop.NativeInterop.Invoker.InvokerNoRet0`1.MarshalAndInvoke(Object delegateContext, ManagedHandle instancePtr, IntPtr paramPtr) in Z:\Flax\Source\FlaxEngine\Source\Engine\Engine\NativeInterop.Invoker.cs:line 213
   at FlaxEngine.Interop.NativeInterop.InvokeMethod(ManagedHandle instanceHandle, ManagedHandle methodHandle, IntPtr paramPtr, IntPtr exceptionPtr) in Z:\Flax\Source\FlaxEngine\Source\Engine\Engine\NativeInterop.Unmanaged.cs:line 675
[ 00:17:53.268 ]: [Error] Exception has been thrown during UICanvas::PostDeserialize.
Invalid ManagedHandl
```
**Flax version:**
1.6.6340

**Comment by mafiesto4** [comment]
It looks like it's related to hot-reload in Editor as I was unable to reproduce it by saving/loading canvas in prefab/scene. But after hot-reload it instantly appears.

**Comment by mafiesto4** [comment]
Fixed in 144b72109b43599bd7cffbf44b8772b06a84b75e
